cmds/dhclient/dhclient.go

```diff
@@ -46,17 +47,27 @@ var (
 )
 
 func ifup(ifname string) (netlink.Link, error) {
-	iface, err := netlink.LinkByName(ifname)
-	if err != nil {
-		return nil, fmt.Errorf("cannot get interface by name %v: %v", ifname, err)
-	}
-
-	if err := netlink.LinkSetUp(iface); err != nil {
-		return nil, fmt.Errorf("%v: %v can't make it up: %v", ifname, iface, err)
-	}
+	start := time.Now().Second()
```

Comment:
Using : tme := time.Now()
and 
for  int(time.Since(tme).Seconds()) < start {

}

worked for me: 

func main() {
	/*
	start := 59
	tmp := 0
	ntmp :=0
	for tmp - start < 30 {
		tmp = time.Now().Second()
		if ntmp != tmp {
			ntmp = tmp
			fmt.Printf("\n%d", ntmp)
		}
		
	}
	*/
	start := 30
	tme := time.Now()
	tmp := 0
	ntmp := 0
	for tmp < start {
		tmp := int(time.Since(tme).Seconds())
			if ntmp != tmp {
				ntmp = tmp
				fmt.Printf("\n%d", ntmp)
		}
	}
}